lib/bots/versionbot.ts

```diff
@@ -833,7 +866,10 @@ export class VersionBot extends ProcBot {
             moddedFiles.push(`CHANGELOG.md`);
 
             // Now we get the new version from the CHANGELOG (*not* the package.json, it may not exist).
-            return exec(`cat ${versionData.fullPath}${_.last(moddedFiles)}`).then((contents: string) => {
+            return cliCommand({
+                command: `cat`,
```

Comment:
Apologies if this is a dumb question, but why use `cat` to get the file's contents, rather than just opening it directly?